src/js/components/LeftPane.js

```diff
@@ -47,60 +132,43 @@ export function LeftPane() {
       onMouseEnter={() => setShowCollapse(true)}
       onMouseLeave={() => setShowCollapse(false)}
     >
-      <InvestigationTitleBar onClearAll={onClearAll} />
-      <div className="investigation-pane-body">
-        <InvestigationHeader
-          view={view}
-          onViewChange={onViewChange}
-          onClearAll={onClearAll}
-        />
+      <DropdownHeader
```

Comment:
It looks like you want to make a re-usable sidebar header component. I think that's a good idea, but it might be better to wait until we have a few instances first. For example, the spaces header will not have a viewSelect dropdown. And maybe a better name for this is "Sidebar header" or "Collapsible Sidebar Header" or CollabsibleSection". Dropdown to me implies a pop menu.

Reply:
The colors and placement look great, but some of the styles didn't make it through.

![image](https://user-images.githubusercontent.com/3460638/84089708-8e93c100-a9a4-11ea-8608-c49d59523499.png)

In this image above, the triangle is 8x8, the folder icon is 11x11, the text is:
```css
font-size: 9px; 
font-weight: 700;
letter-spacing: 0.8px;
```

If that's not what was in the design let me know. Maybe you had an old version.